renovate.json

```diff
@@ -54,7 +54,7 @@
       "fileMatch": ["\\.cs$"],
       "matchStringsStrategy": "any",
       "matchStrings": [
-        "WithImage(\"(?<imageName>.*?):(?<version>.*?)\")"
+        "(?<depName>workleap\\/eventgridemulator):(?<currentValue>[-a-z0-9.]*)"
```

Comment:
I would go with a more specific regex for the value
```suggestion
        "(?<depName>workleap\\/eventgridemulator):(?<currentValue>[0-9]+\.[0-9]+\.[0-9]+(-[-a-zA-Z0-9.]+)?)"
```

Reply:
Using `[0-9]+\\.[0-9]+\\.[0-9]+(-[-a-zA-Z0-9.]+)?` to avoid linting errors within VS Code. However, I tested that both regexes work

With double \\
![image](https://github.com/gsoft-inc/wl-domain-event-propagation/assets/158102624/93dd6af6-df4d-44af-81b6-f093246bc283)

With single \
![image](https://github.com/gsoft-inc/wl-domain-event-propagation/assets/158102624/170858d0-165e-4dd9-9d3a-cd737887856a)